### ISSUE TITLE
wrong rules regarding role="presentation"

### ISSUE THREAD

**Issue by chris-scheurle** [validator/validator#520]
#### Steps to reproduce:

Go to [http://validator.w3.org/nu/#textarea](http://validator.w3.org/nu/#textarea), enter the following code:
```html
<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
<p>Some text</p>
<hr role="presentation">
<dl role="presentation">
<dt>1</dt>
<dd>one</dd>
</dl>
</body>
</html>
```

Validator shows 2 error messages: 
> Bad value `presentation` for attribute `role` on element _[...]_

But the spec (see links enclosed in the error messages) explicitly allows that role for those elements.

**Comment by stevefaulkner** [comment]
@sideshowbarker, @chris-scheurle is correct - role=presentation allowed on both:

- http://w3c.github.io/html-aria/#dl
- http://w3c.github.io/html-aria/#hr
